server/anoixo-server/text_providers/Nestle1904LowfatProvider.py

```diff
@@ -262,7 +262,17 @@ def _build_query_string(self, query: TextQuery) -> str:
                 return map {{
                   "text": local:punctuated($w),
                   "matchedSequence": {matched_sequence_switch},
-                  "matchedWordQuery": {matched_word_query_switch}
+                  "matchedWordQuery": {matched_word_query_switch},
+                  "class": data($w/@class),
```

Comment:
Can you generate this using the list of allowedAttributes instead of hardcoding each one?